packages/shared/src/components/post/PostActions.tsx

```diff
@@ -150,6 +192,48 @@ export function PostActions({
           >
             Copy
           </QuaternaryButton>
+          {!!post.author && (
+            <ConditionalWrapper
+              condition={post?.userState?.awarded}
+              wrapper={(children) => {
+                return (
+                  <SimpleTooltip content="You already awarded this post!">
+                    <div>{children}</div>
+                  </SimpleTooltip>
+                );
+              }}
+            >
+              <QuaternaryButton
+                id="award-post-btn"
+                pressed={post?.userState?.awarded}
+                onClick={() => {
+                  if (!user) {
+                    return showLogin({ trigger: AuthTriggers.GiveAward });
+                  }
+
+                  return openModal({
+                    type: LazyModal.GiveAward,
+                    props: {
+                      type: 'POST',
+                      entity: {
+                        id: post.id,
+                        receiver: post.author,
+                        numAwards: post.numAwards,
+                      },
+                    },
+                  });
+                }}
+                icon={<MedalBadgeIcon />}
+                responsiveLabelClass={actionsClassName}
+                className={classNames(
+                  'btn-tertiary-cabbage',
```

Comment:
Why does it need this class.
Button should also accept color right?

Reply:
It is how other buttons were made so I just did the same thing